src/templates/ui-grid/ui-grid-header.html

```diff
@@ -25,4 +25,18 @@
       </div>
     </div>
   </div>
+  <div ng-if="typeof(grid.options.showTopFooter) != 'undefined' && grid.options.showTopFooter"
```

Comment:
Thanks. I've been around JS for decades, but new to angular. I tend to be a bit anal with typeof to make sure I'm not asking for something that hasn't been declared yet, but it's nice to know that angular automatically traps this and I don't have to worry about it in an ng-if scenario.